### PR TITLE
Burger menu

### PR DESCRIPTION
**Pull request by sylcym**

### CODE REVIEW

File: src/Components/Nav/Nav.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { Navbar } from '../Navbar/Navbar';
+import './Nav.scss';
+
+export const Nav = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <nav className="nav">
+      <div className="nav_btns">
```
Comment (DanielSorokowski) [inline src/Components/Nav/Nav.tsx:10]: nav__btsn

File: src/Components/Nav/Nav.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <button
+          className={`nav_burger nav_burger--open `}
```
Comment (DanielSorokowski) [inline src/Components/Nav/Nav.tsx:12]: look for single "_" in your code

File: src/Components/Nav/Nav.tsx (hunk 1 above)
Comment (sylcym) [reply src/Components/Nav/Nav.tsx]: done

File: src/Components/Nav/Nav.tsx (hunk 2 above)
Comment (sylcym) [reply src/Components/Nav/Nav.tsx]: done

**Pull request merged**